src/Featurizers/ForecastingPivotFeaturizer.h

```diff
@@ -0,0 +1,129 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#pragma once
+
+#include "../Archive.h"
+#include "../Traits.h"
+
+#include "Components/InferenceOnlyFeaturizerImpl.h"
+
+namespace Microsoft {
+namespace Featurizer {
+namespace Featurizers {
+
+/////////////////////////////////////////////////////////////////////////
+///  \class         ForecastingPivotTransformer
+///  \brief         Expand output from upstream featurizers where all values
+///                 in the same column across matrixes are not null/empty
+///
+template <typename InputT>
+class ForecastingPivotTransformer :
+    public Components::InferenceOnlyTransformerImpl<
+        typename std::vector<Microsoft::Featurizer::RowMajMatrix<typename Microsoft::Featurizer::Traits<InputT>::nullable_type>>,
+        std::vector<typename Microsoft::Featurizer::Traits<InputT>::nullable_type>
```

Comment:
>typename Microsoft::Featurizer::Traits<InputT>::nullable_type [](start = 20, length = 61)

I think that this can be `InputT` because we will only output rows where all matrix elements are not empty.

Reply:
Good point! Changed to `InputT`.